extensions/amp-web-push/0.1/validator-amp-web-push.protoascii

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-web-push
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-web-push extension .js script"
+  extension_spec: {
+    name: "amp-web-push"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-web-push>
+  html_format: AMP
+  tag_name: "amp-web-push"
+  requires: "amp-web-push extension .js script"
```

Comment:
apologies, `requires` has been replaced by `requires_extension` which in this case would be `requires_extension: "amp-web-push"`.